types/environment.go

```diff
@@ -2,11 +2,11 @@ package types
 
 type Environment struct {
 	IdModel
+	Type           string         `json:"type"`
 	Name           string         `json:"name"`
 	Reference      string         `json:"reference"`
 	OrgName        string         `json:"orgName"`
 	StackId        int64          `json:"stackId"`
-	StackName      string         `json:"stackName"`
 	ProviderConfig ProviderConfig `json:"providerConfig"`
-	PipelineOrder  int            `json:"pipelineOrder"`
+	PipelineOrder  *int           `json:"pipelineOrder"`
```

Comment:
This will break the CLI when this library is updated.

Reply:
I have added a new PR for nullstone repo to address this.